tests/testthat/test-transfermarkt.R

```diff
@@ -11,24 +11,58 @@ test_that("tm_player_market_values() works", {
 
 })
 
+test_that("tm_player_transfer_history() works", {
+  testthat::skip_if_offline()
+  testthat::skip_on_cran()
+
+  ## multiple URLs
+  transfer_data <- tm_player_transfer_history(
```

Comment:
augmented the test a bit while i was at it.

- explicit testing for multiple URLs
- explicit testing of `get_extra_info`
- checks for exact column names (this should be less frail than what we used to do for fotmob since we explicitly define the column names in the function code)